Integrations/integration-PostgreSQL.yml

```diff
@@ -84,7 +84,7 @@ script:
             table.append(tableRow)
 
         demisto.results( {'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': contents, "ReadableContentsFormat": formats['table'], "HumanReadable": table} );
-
+        conn.close()
```

Comment:
You should also do cur.close() before

![image](https://user-images.githubusercontent.com/5429033/36395137-2d45ae60-15c0-11e8-9236-794dfddf4a2c.png)